ios/FluentUI/Command Bar/CommandBarButtonGroupView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class CommandBarButtonGroupView: UIView {
+    let buttons: [CommandBarButton]
+
+    init(buttons: [CommandBarButton]) {
+        self.buttons = buttons
+
+        super.init(frame: .zero)
+        translatesAutoresizingMaskIntoConstraints = false
+
+        clipsToBounds = true
+        layer.cornerRadius = CommandBarButtonGroupView.cornerRadius
+        if #available(iOS 13, *) {
+            layer.cornerCurve = .continuous
+        }
+
+        configureHierarchy()
+        applyInsets()
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private lazy var stackView: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: buttons)
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .horizontal
+        stackView.spacing = CommandBarButtonGroupView.buttonPadding
+
+        return stackView
+    }()
+}
+
+private extension CommandBarButtonGroupView {
```

Comment:
why use an additional extension?

Reply:
To group private members together.

Reply:
extensions in `FluentUI` are preserved for grouping functionality as opposed to access levels